extensions/pyRevitTools.extension/checks/audit_all_check.py

```diff
@@ -0,0 +1,396 @@
+            "Reference Planes", 
+            card_builder(100, reference_planes_count, ' Ref Planes'), card_builder(10, unnamed_ref_planes_count, ' Ref Planes no_name')
+            )
+
+        html_content = body_css + \
+            critical_elements_frame + \
+            rooms_frame+ \
+            sheets_views_graphics_frame + \
+            cad_files_frame + \
+            families_frame + \
+            graphical2d_elements_frame + \
+            groups_summary_frame + \
+            reference_planes_frame + \
+            create_frame(
+                'Materials',
+                card_builder(100, materials_count, ' Materials')
+                )
+
+        output.print_html(html_content + '</div>')
+
+        # csv export
+        data = [user, date, doc_clean_name, revit_version_build, project_name, project_number, project_client, project_phases, worksets_names, element_count, purgeable_elements_count, all_warnings_count, critical_warnings_count, rvtlinks_count, activated_analytical_model_elements_count, rooms_count, unplaced_rooms_count,unbounded_rooms, sheets_count, views_count, views_not_on_sheets, schedule_count, schedules_not_sheeted_count, copied_views_count, view_templates_count, unused_view_templates_count, all_filters_count, unused_view_filters_count, materials_count, line_patterns_count, dwgs_count, linked_dwg_count, inplace_family_count, not_parametric_families_count, family_count, imports_subcats_count, generic_models_types_count, detail_components_count, text_notes_types_count, text_bg_count, text_notes_types_wf_count, text_notes_count, text_notes_caps_count, detail_groups_count, detail_groups_types_count, model_group_count, model_group_type_count, reference_planes_count, unnamed_ref_planes_count, detail_lines_count, dim_types_count, dim_count, dim_overrides_count, revision_clouds_count]
+        data_str = [str(i) for i in data]
+
+        export_to_csv(doc_clean_name, data, data_str, output)
+
+        # RVTLinks
+        if rvtlinks_elements_items:
+            generate_rvt_links_report(output, rvtlinks_docs, body_css)
+        output.self_destruct(180)
+    except Exception as e:
+        print(format_exc())
+        print(e)
+
+
+def generate_rvt_links_report(output, rvtlinks_docs, body_css):
+    output.print_md('# RVTLinks')
+    for rvtlink in rvtlinks_docs:
+        link_name = doc_name(rvtlink)
+        link_printed_name = clean_name(rvtlink)
+        output.print_md('## ' + link_printed_name)
+        output.print_md('___')
+        if rvtlink is None:
+            continue
+        output.print_md(link_name)
+        element_count = elements_count(rvtlink)
+        purgeable_elements_count = get_purgeable_count(rvtlink)
+        all_warnings_count, _, warnings_guid = warnings(rvtlink)
+        critical_warnings_count = critical_warnings(warnings_guid,CRITICAL_WARNINGS)
+        rvtlinks_elements_items, rvtlinks_count, type_status, rvtlinks_docs = rvtlinks_elements(rvtlink)

```

Comment:
here you're rewriting the `rvtlinks_docs` list that is used in the for loop, is this the intended behavior?
I believe the quickest solution is to rename the function argument and line 241 to something else